tools/ldb_cmd_test.cc

```diff
@@ -262,6 +263,7 @@ class FileChecksumTestHelper {
         return cs;
```

Comment:
Nit: better to also re-enable on error path also

Reply:
Fixed.